serverless.yml

```diff
@@ -132,7 +132,7 @@ functions:
           startingPosition: LATEST
 
   updateDrillStatus:
-    handler: stopcovid/drill_progress/aws_lambdas/update_drill_status.handler
```

Comment:
This lambda has been repurposed to just publish dialog event batches to kinesis

Reply:
So these go to a kinesis stream that the new app consumes?

Reply:
Correct @pdsarin. Two main reasons:

1) Having only two consumers of the dynamo stream was going to be a problem during the migration
2) There isn't a great way to consume dynamo streams in python. I wrote a custom one modeled off of the kinesis consumer and it was pretty tricky, mainly due to the ephemerality of dynamo stream shards